pkg/kubelet/container/helpers.go

```diff
@@ -91,7 +91,7 @@ func ShouldContainerBeRestarted(container *v1.Container, pod *v1.Pod, podStatus
 // HashContainer returns the hash of the container. It is used to compare
 // the running container with its desired spec.
 func HashContainer(container *v1.Container) uint64 {
-	hash := adler32.New()
+	hash := fnv.New32a()
```

Comment:
Users can opt-out CRI for this release, but they can't turn this off though. We should use the new hash for CRI, but the old adler for dockertools. This way, users who don't want to upgrade won't be affected, and new CRI users will get it anyway.